src/server/bin.js

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
```

Comment:
permissions should be 755 (are 644 right now, which is missing x bit)

Reply:
Good catch!